server/schema/mutations.js

```diff
@@ -71,12 +71,18 @@ const mutation = new GraphQLObjectType({
         body: {type: GraphQLString},
         tags: {type: GraphQLList(GraphQLString)}
       },
-      resolve(parentValue, {title, body, tags}, context) {
-        if (!context.user) {
```

Comment:
why did you remove context here?

Reply:
At the time of writing the code i thought it would be easier to understand

Reply:
@nashawn-griffith context is a graphql standard thing for authentication. I see that you didn't change it in the server.js file, so did you destructure user from context somewhere? Am not seeing that anywhere.